src/client/Microsoft.Identity.Client/Internal/Requests/Silent/BrokerSilentStrategy.cs

```diff
@@ -79,7 +79,9 @@ private async Task<MsalTokenResponse> SendTokenRequestToBrokerAsync()
                 _logger.Info(
                     LogMessages.ErrorReturnedInBrokerResponse(msalTokenResponse.Error));
 
-                if (msalTokenResponse.Error == BrokerResponseConst.NoTokenFound)
+                if (msalTokenResponse.Error == BrokerResponseConst.NoTokenFound ||
```

Comment:
These error codes are Android specific and I think it would be better for the AndroidBroker to handle them, i.e. I would move this logic to https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/93c3a6893477e0ee3ef3c0b63db94b8e1aca6e4c/src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBroker.cs#L109